joomla/joomla-cms
# [4.0] Hide Mass Mail Users menu when Send Mail is disabled

Pull Request for Issue #25508 .

### Summary of Changes
Hide `Mass Mail Users` menu when `Send Mail` is disabled which takes precedence of `Disable Mass Mail`.


### Testing Instructions
For screenshots, go to #25508 

Go Global Configuration > Server
Set `Send Mail` to Yes
Set `Disable Mass Mail` to No
Click Save
Under `Users` menu, `Mass Mail Users` is displayed
Set `Send Mail` to No
Click Save
Under `Users` menu, `Mass Mail Users` should not be displayed
